src/conversion.c

```diff
@@ -405,11 +405,12 @@ int igraph_to_directed(igraph_t *graph,
         IGRAPH_CHECK(igraph_create(&newgraph, &edges,
                                    (igraph_integer_t) no_of_nodes,
                                    IGRAPH_DIRECTED));
-        IGRAPH_FINALLY(igraph_destroy, &newgraph);
-        igraph_vector_destroy(&edges);
+        IGRAPH_FINALLY(igraph_destroy, &newgraph);        
         IGRAPH_I_ATTRIBUTE_DESTROY(&newgraph);
         IGRAPH_I_ATTRIBUTE_COPY(&newgraph, graph, 1, 1, 1);
```

Comment:
Should this also be enclosed in `IGRAPH_CHECK`?

Reply:
No, it's a macro, and has no return code (however, it does include an IGRAPH_ERROR, so there is a possibility of failure).